app/assets/javascripts/bastion/layouts/one-column-details.html

```diff
@@ -0,0 +1,6 @@
+<div class="row" ng-hide="page && (page.loading || page.error)">
+  <section class="col-sm-12">
+    <div data-block="left-column"></div>
```

Comment:
Very minor comment, should we even mention left/right columns in this case?  Also, should this even be called "one-column" since it's really just a normal page?